cdap-security/src/test/java/co/cask/cdap/security/authorization/ExceptionAuthorizationEnforcer.java

```diff
@@ -43,7 +44,7 @@ public void enforce(EntityId entity, Principal principal, Set<Action> actions) t
 
   @Override
   public Set<? extends EntityId> isVisible(Set<? extends EntityId> entityIds, Principal principal) throws Exception {
-    throw new UnsupportedOperationException("This method needs to be implemented!");
+    return Collections.emptySet();
```

Comment:
Looks like all the other methods are throwing exception, shouldn't this also throw an exception?